catlas/filter_utils.py

```diff
@@ -75,13 +122,12 @@ def get_pourbaix_stability(mpid: str, conditions: dict) -> list:
     str_en = mpid.encode("ascii")
     txn = env.begin()
     getit = txn.get(str_en)
-    if getit == None:
-        env.close()
+    entry = pickle.loads(getit)
```

Comment:
this is very hard to read; can you add some comments to explain what you're doing?